modules/configuring-your-ide.adoc

```diff
@@ -70,7 +70,7 @@ ifdef::cloud-service[]
 [IMPORTANT] 
 ====
 *Disclaimer:* 
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms. You should review their licensing terms before you use this sample workbench.
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[https://rstudio.org/] and is subject to their licensing terms. You should review their licensing terms before you use this sample workbench.
```

Comment:
I checked the link and it's fine. However, would this be a good time to remove the word "should"?

Reply:
Agreed - updated to remove.